src/components/Incorporation/UploadRules.vue

```diff
@@ -111,7 +111,7 @@
                   <v-col cols="1"><v-icon>mdi-circle-small</v-icon></v-col>
                   <v-col cols="11">
                     <p class="mb-0">
-                      The Cooperative name is identified <b>exactly</b> as follows throughout the Memorandum:
+                      The Cooperative name is identified <b>exactly</b> as follows throughout the Rules of the Association:
```

Comment:
Please run lint locally. There is a warning here that you need to fix.

![image](https://user-images.githubusercontent.com/10002889/163475288-04b0705e-12d8-45e6-9566-0a62596a87fe.png)

(Sometimes our GH CI action catches these and sometimes not. But it's good practise to fix it if you broke it :)  )

Reply:
I think it's gone now. It was a warning from eslint that line 114 was over the limit of 120 characters.

Reply:
This line still appears to be 124 columns wide in this PR. Did you change it and commit the change?

Reply:
I am using a private variable for the long string and use disable eslint from there. This line of code is outdated. However, I am not sure if that's going to work, because I was unable to access the Incorporation Application.

Reply:
I don't see this changed file any more. Maybe you clobbered it during a rebase or squash.

I saw your changes temporarily. They should work. You'll need to create the right kind of IA to test it.

PS Use eslint disable sparingly. I'd prefer to see a long line broken up into 2 lines.

Reply:
Here's a link to your commit: https://github.com/bcgov/business-create-ui/commit/f783a75c1cf34096a94c6f1497d159e1442cd616

You should be able to cherry-pick it and then push it to GH.

Reply:
I lost it somewhere, and it should be there now. Thanks a lot.

Reply:
The file is back! :)

Would you mind breaking up your long line up into 2 lines, so you don't have to use eslint-disable? If it looks ugly, consider just having the text on 2 lines in the template...?

Reply:
Sure. I broke the this long line into 2 lines in my latest commit.